crates/pep508-rs/src/lib.rs

```diff
@@ -752,6 +766,35 @@ fn parse_url(cursor: &mut Cursor, working_dir: Option<&Path>) -> Result<Verbatim
     Ok(url)
 }
 
+/// Parse a filesystem path from the [`Cursor`], advancing the [`Cursor`] to the end of the path.
+///
+/// Returns `false` if the path is not a clear and unambiguous filesystem path.
+fn looks_like_file_path(cursor: &mut Cursor) -> bool {
+    let Some((_, first_char)) = cursor.next() else {
+        return false;
+    };
+
+    // Ex) `/bin/ls`
+    if first_char == '\\' || first_char == '/' || first_char == '.' {
```

Comment:
Does this support `pip install path/to/my.whl`? I suspect that this is a big fraction of the cases

Reply:
Yeah, that already works on main, because “path” is parsed as the package name, then we fail on the slash. The case covered in this PR is when there’s no leasing alpha character (like “./path”).